TeacherCourse/models.py

```diff
@@ -0,0 +1,70 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+Category = [
+    ('BUSINESS_AND_MANAGEMENT', 'Business & Management'),
+    ('COMPUTER_SCIENCE_AND_IT', 'Computer Science & IT'),
+    ('CREATIVE_ARTS', 'Creative Arts'),
+    ('ECONOMICS', 'Economics'),
+    ('HISTORY', 'History'),
+    ('HEALTH_AND_PSYCHOLOGY', 'Health & Psychology'),
+    ('LITERATURE', 'Literature'),
+    ('LAW', 'Law'),
+    ('LANGUAGES_AND_CULTURES', 'Languages & Cultures'),
+    ('MATHS', 'Maths'),
+    ('MARKETING', 'Marketing'),
+    ('POLITICS_AND_SOCITY', 'Politics & Socity'),
+    ('SCIENCE_AND_ENGINEERING', 'Science & Engineering'),
+    ('SPORT_AND_LEISURE', 'Sport & Leisure'),
+]
+
+
+class TeacherCoursesManager(models.Manager):
+
+    def in_price_range(self, min, max):
```

Comment:
I suggest using type hints in functions signatures, I think its more readable this way.
So  here, it would be something like:
`def in_price_range(self, min: int, max: int):`